app/en/home/build-tools/secure-your-mcp-server/page.mdx

```diff
@@ -0,0 +1,480 @@
+---
+title: "Adding Resource Server Authentication"
+description: "Secure your HTTP MCP server with OAuth 2.1 Resource Server authentication"
+---
+
+import { Steps, Tabs, Callout } from "nextra/components";
+
+# Adding Resource Server Authentication to Your MCP Server
+
```

Comment:
This is rather technical - I think we should start with something a little higher level (so... you want to deploy this thing, but need to protect it...).  I think you can steal from https://docs.arcade.dev/en/home/serve-tools/securing-arcade-mcp